cmd/tailscale/cli/up.go

```diff
@@ -294,6 +296,29 @@ func runUp(ctx context.Context, args []string) error {
 		},
 	}
 
+	if upArgs.json {
+		opts.Notify = func(n ipn.Notify) {
+			if n.ErrMessage != nil {
+				fmt.Fprintf(os.Stdout, `{"error":%q}\n`, *n.ErrMessage)
```

Comment:
these newlines in raw string literals aren't what you mean. did your push to fix that come during github's outage earlier? push again?